Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/di/ApplicationComponent.kt

```diff
@@ -70,7 +71,8 @@ import javax.inject.Singleton
         BugReportingModule::class,
         SerializationModule::class,
         WorkerBinder::class,
-        ContactDiaryModule::class
+        ContactDiaryModule::class,
+        ContactDiaryCleanTaskModule::class
```

Comment:
Would prefer this do be included via `ContactDiaryModule`, so the whole diary stuff is included via a single statement.